### DISCUSSION TITLE
Var type values

### DISCUSSION THREAD

**Discussion by ElcomJ** [neoclide/coc.nvim#4105] [Q&A] [answered]
Good afternoon, I wonder if there is any highlight to be able to change the background color of this selected data, or if they would have a way to disable them.

![image](https://user-images.githubusercontent.com/37147897/186753470-94cc10ce-e2ef-4440-af45-276383478db7.png)

Thank you in advance.

**Comment by fannheyward** [comment, marked as answer]
The hi group is `CocInlayHint`.

**Comment by chemzqm** [comment]
`:h coc-inlayHint`
